pkg/packetfilter/nftables/nftables.go

```diff
@@ -411,7 +414,7 @@ func FromRuleSpec(spec RuleSpec) *packetfilter.Rule {
 				rule.SnatCIDR, i = parseNextTerm(spec, i, noopParse)
 			}
 		case "mark":
-			rule.MarkValue, i = parseNextTerm(spec, i, noopParse)
+			rule.MarkValue, i = parseMark(spec, i)
```

Comment:
Perhaps it would be better to check for case "meta" and then check the next term for "mark". Same with "set" above. That would make both consistent simplify `parseMark`.

Reply:
bump

Reply:
We need to parse: 
a. counter meta mark set mark or 0xc000
b. mark and 0xc000 == 0xc0000

I updated 'parseAction' to handle 'counter meta mark set' for mark action, and the case "mark" here should parse mark value for both cases.

I think it is OK.